norms.md

```diff
@@ -0,0 +1,22 @@
+##Fishtown Analytics Norms##
+
+###Listen, then speak.###
```

Comment:
This is the only thing you've written so far that I don't actually agree with. I think "think, talk, think" and "talk, think, talk" are both personal styles and ones that we should not attempt to create norms around. I do agree with the first sentence of this paragraph though.